homeassistant/components/plugwise/sensor.py

```diff
@@ -138,13 +129,13 @@
     "gas_consumed_interval": [
         "Current Consumed Gas",
         VOLUME_CUBIC_METERS,
-        DEVICE_CLASS_GAS,
+        None,
```

Comment:
Let's not use None but "gas".

Reply:
@bdraco please advise - was your comment not to declare the constant - or to leave it out (i created issue 388 on architecture accordingly.

Reply:
Please leave it out since we can only use device classes that the defined by Home Assistant.

Reply:
Ok, clear, closing those remarks.